apps/watch/src/components/CollectionsPage/ContainerHero/CollectionsHeader/LanguageModal.tsx

```
@@ -0,0 +1,106 @@
+              >
+                {language.name}
+              </a>
+            ))}
+
+            <a
+              href={`/contact`}
+              className="mt-6 inline-block text-center gap-2 px-4 py-4 text-md text-white font-semibold tracking-wider rounded-lg bg-stone-800/10 hover:bg-red-500 hover:text-white transition-colors duration-200 cursor-pointer"
+              data-testid={`language-button-support`}
+            >
+              <EmailIcon sx={{ fontSize: 20, marginRight: 2 }} /> Give Feedback
+            </a>
```

ESLint

[error] disallow literal string: <a
              href={`/contact`}
              className="mt-6 inline-block text-center gap-2 px-4 py-4 text-md text-white font-semibold tracking-wider rounded-lg bg-stone-800/10 hover:bg-red-500 hover:text-white transition-colors duration-200 cursor-pointer"
              data-testid={`language-button-support`}
            >
              <EmailIcon sx={{ fontSize: 20, marginRight: 2 }} /> Give Feedback
            </a>

(i18next/no-literal-string)